src/openforms/js/components/admin/form_design/PaymentFields.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {FormattedMessage} from 'react-intl';
+
+import Field from '../forms/Field';
+import FormRow from '../forms/FormRow';
+import Fieldset from '../forms/Fieldset';
+import Select from "../forms/Select";
+import Form from "@rjsf/core";
+
+
+const PaymentFields = ({
+    backends=[],
+    selectedBackend='',
+    backendOptions={},
+    onChange
+}) => {
+    const backendChoices = backends.map( backend => [backend.id, backend.label]);
+    const backend = backends.find( backend => backend.id === selectedBackend );
+    const hasOptionsForm = Boolean(backend && Object.keys(backend.schema.properties).length);
+
+    return (
+        <Fieldset>
+            <FormRow>
+                <Field
+                    name="form.paymentBackend"
+                    label={<FormattedMessage description="Payment backend label" defaultMessage="Select payment backend" />}
+                >
+                    <Select
+                        choices={backendChoices}
+                        value={selectedBackend}
+                        onChange={ (event) => {
+                            onChange(event);
+                            // Clear options when changing backend
+                            onChange({target: {name: 'form.paymentBackendOptions', value: {}}})
+                        }}
+                        allowBlank
+                    />
+                </Field>
+            </FormRow>
+
+            {
+                hasOptionsForm
+                ? (
+                    <FormRow>
+                        <Field
+                            name="form.paymentBackendOptions"
+                            label={<FormattedMessage description="Payment backend options label" defaultMessage="Payment backend options" />}
+                        >
+                            <Form
```

Comment:
I find the combination of `Field` and the `Form` from rjsf causes the help text to render a bit strange:
![Screenshot from 2021-08-17 10-12-42](https://user-images.githubusercontent.com/19154114/129689269-32e3ca0e-5a18-4e96-9b98-f0b34f6518fc.png)

Reply:
It is - I looked if there was an easy CSS fix but there was not, and altering the layout didn't look straight forward either, so I decided to tackle that later.